.github/workflows/check-sources.yml

```diff
@@ -0,0 +1,10 @@
+name: Sources Check
+on: [pull_request]
+jobs:
+  check-sources:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: ./scripts/check-cla
+      - run: ./scripts/check-lint
```

Comment:
`check-lint` should be in a separate workflow, because it should also run on pushes to master. `check-cla` however makes sense only for pull requests.